src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -338,6 +338,11 @@
       <MissingIgnoredAssemblies Include="@(IgnoreVsixFilesList)" Exclude="@(AllVsixAssemblies)" />
       <VSIXSourceItem Remove="@(SuppressPackaging->'%(SourceFile)')" />
       <VSIXSourceItem Remove="@(AllVsixSymbols->'%(SourceFile)')" />
+      <!-- Don't get Lucene.Net from a project reference -->
+      <VSIXSourceItem Remove="@(VSIXSourceItem)" Condition="'%(Filename)%(Extension)' == 'Lucene.Net.dll' And '%(VSIXSourceItem.ReferenceSourceTarget)' == 'ProjectReference'" />
```

Comment:
Let's add Lucene.Net to the ignore list with a comment saying "ignore the Lucene.Net that the VS SDK adds because this project will manually add the signed one"

Also, remove the Remove below (lines 344-355).  Looks like this whole file can be undone technically.

Reply:
Ok, I've ignored Lucene.net, but I'm not following how we can remove the change we made together offline to remove the VSIXItems for already added .resources.dlls. How would that work now?

Reply:
Talked offline and we can ignore Lucene.net from this calculation since elsewhere in the build we add it from `GetThirdPartyAssemblies`.